app/models/file_node.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 class FileNode < Valkyrie::Model
+  include Valkyrie::Model::AccessControls
```

Comment:
Fair enough. The loose ends will not be too far or wide if this should be changed.